### PR TITLE
Add Django Rest Framework

### PR DESCRIPTION
**Pull request by klemmari1**
- Modify the pysocial url path.
- Add Django Rest Framework.
- Add models for Hitas and Haso application forms.
- Add serializers for Hitas and Haso applications.
- Add viewsets for Hitas and Haso applications.
- Add CRUD tests for both the HASO and HITAS endpoints.

Refs ASU-285 ASU-354 ASU-355

### CODE REVIEW

File: application_form/tests/test_haso_application.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    response = api_client.put(
+        reverse("v1:hasoapplication-detail", kwargs={"pk": haso_application.id}), data
+    )
+
+    assert response.status_code == 200
+    assert response.data["is_changing_occupancy_apartment"] is False
+
+
+@pytest.mark.django_db
+def test_haso_applicaiton_delete(api_client):
```
Comment (ehaivala) [inline application_form/tests/test_haso_application.py:66]: All test function names in this module have typo in the word "application".

File: application_form/tests/test_hitas_application.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    response = api_client.put(
+        reverse("v1:hitasapplication-detail", kwargs={"pk": hitas_application.id}), data
+    )
+
+    assert response.status_code == 200
+    assert response.data["has_children"] is False
+
+
+@pytest.mark.django_db
+def test_hitas_applicaiton_delete(api_client):
```
Comment (ehaivala) [inline application_form/tests/test_hitas_application.py:59]: Same typo in the test function names in this module as well.

**Review by ehaivala — CHANGES_REQUESTED** [review]
Looks like a good starting point. After the typos are fixed this is good to go. 👍

File: application_form/tests/test_haso_application.py (hunk 1 above)
Comment (klemmari1) [reply application_form/tests/test_haso_application.py:66]: Good spot. I'll fix :+1:

**Review by ehaivala — APPROVED** [review]
Nice work 👍

**Pull request merged**